### PR TITLE
Translate dev/js/tensorflow.md to zh #1113 finished by dz

### PR DESCRIPTION
**Pull request by dreammyboy**
Signed-off-by: Dong Zhe <dreammyboy@163.com>

### CODE REVIEW

File: docs/book/zh/src/dev/js/tensorflow.md [hunk 1]
```diff
@@ -1 +1,80 @@
...
+}
+label_file.close();
+
+print('label:');
+print(label);
+print('confidence:');
+print(max / 255);
+```
+
+为了在 WasmEdge runtime 中运行 JavaScript，你可以通过在 CLI 中执行如下操作，这样就可以重新构建一个包含 TensorFlow 的 QuickJS 引擎，然后就可以在 JavaScript 应用中调用 TensorFlow API 了。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/tensorflow.md:44]: 把 通过 去掉？

File: docs/book/zh/src/dev/js/tensorflow.md [hunk 2]
```diff
@@ -1 +1,80 @@
...
+... ...
+$ cd example_js/tensorflow_lite_demo
+$ wasmedge-tensorflow-lite --dir .:. ../../target/wasm32-wasi/release/wasmedge_quickjs.wasm main.js
+label:
+Hot dog
+confidence:
+0.8941176470588236
+```
+
+> 注意: 命令行中的 `--dir .:.` 是为了给wasmedge开启本地文件读取权限， 以便正常读取到 `main.js` 文件.
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/tensorflow.md:57]: wasmedge 前后空一格

**Review by alabulei1 — DISMISSED** [review]
Great! Thank you for your contribution.

**Comment by dreammyboy** [comment]
Hi，已经按建议修改了，请再review一下哈 :)

**Comment by alabulei1** [comment]
LGTM! @hydai  Could you help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**